packages/snaps-execution-environments/src/common/endowments/network.ts

```diff
@@ -121,9 +171,26 @@ const createNetwork = () => {
       );
     }
 
+    let started = false;
+    const onStart = () => {
+      if (!started) {
+        started = true;
+        notify({ method: 'OutboundRequest' });
```

Comment:
I'm wondering if this should have some identifier that would help us identify from where this `OutboundRequest` is coming 🤔 

Might be useful for future maintenance or debugging if needed. 

I feel like now `fetch` has more exclusive access to the platform's communication channels. Somehow `fetch` is still just an endowment which is attenuated in some or many ways, contained and controlled within the Sandbox (SES compartment) and with this it's getting a little bit outside of that "world".

Reply:
See https://github.com/MetaMask/snaps/pull/1756#discussion_r1379726705

It does add some coupling between fetch and our communication channels, but I don't see a way to go about this without increasing the coupling. We also already have that coupling for ``snap.request` and `ethereum.request`

Reply:
Adding params to identify it is fair enough.